Buildings/Resources/Scripts/Dymola/Media/Examples/SteamDerivativeCheck.mos

```diff
@@ -0,0 +1,15 @@
+﻿simulateModel("Buildings.Media.Examples.SteamDerivativeCheck", startTime=0, stopTime=1, tolerance=1E-10, method="dassl", resultFile="MediaExamplesSteamDerivativeCheck");
```

Comment:
I believe "cvode" is now the default solver for MBL. Not sure if it applies to the steam related models.

Reply:
Changed to CVODE.